e2e/framework/clients/k8s_rook_block.go

```diff
@@ -0,0 +1,138 @@
+package clients
+
+import (
+	"fmt"
+	"github.com/rook/rook/e2e/framework/contracts"
+)
+
+type k8sRookBlock struct {
+	transportClient contracts.ITransportClient
+}
+
+var (
+	listCmd              = []string{"rook", "block", "ls"}
+	writeDataToBlockPod  = []string{"sh", "-c", "WRITE_DATA_CMD"}
+	readDataFromBlockPod = []string{"cat", "READ_DATA_CMD"}
+)
+
+// Constructor to create k8sRookBlock - client to perform rook Block operations on k8s
+func CreateK8SRookBlock(client contracts.ITransportClient) *k8sRookBlock {
+	return &k8sRookBlock{transportClient: client}
+}
+
+// Function to create a Block using Rook
+// Input paramaters -
+//name - path to a yaml file that creates a pvc in k8s - yaml should describe name and size of pvc being created
+//size - not user for k8s implementation since its descried on the pvc yaml definition
+//Output - k8s create pvc operation output and/or error
+func (rb *k8sRookBlock) BlockCreate(name string, size int) (string, error) {
+	cmdArgs := []string{name}
+	out, err, status := rb.transportClient.Create(cmdArgs, nil)
+	if status == 0 {
```

Comment:
yes,  we have seen this a few times, we will make sure there is no error and them even when status is 0. We have to update this in multiple places so we will update this in .5. i will create a ticket.